content/articles/ldnpydojos07e02.md

```diff
@@ -0,0 +1,175 @@
+Title: The London Python Dojo S07E02
+Date: 2015-10-03
+Short_summary: The We Are Wizards team was at the last London Python Code Dojo.
+Category: Events
+Authors: Gautier
+
+
+The We Are Wizards team was present at the last London Python Code Dojo held at
+Bank of America in Canary Wharf on the first of October.
```

Comment:
This sounds incredibly formal :)

Maybe We (wizards) attended the London ... at BAML at .. . Find our report inside.

?